time/src/main/java/net/jqwik/time/DefaultMonthArbitrary.java

```diff
@@ -0,0 +1,57 @@
+package net.jqwik.time;
+
+import java.time.*;
+
+import net.jqwik.api.*;
+import net.jqwik.api.arbitraries.*;
+import net.jqwik.api.time.*;
+
+public class DefaultMonthArbitrary extends ArbitraryDecorator<Month> implements MonthArbitrary {
+
+	private Month min = Month.JANUARY;
+	private Month max = Month.DECEMBER;
+	private Month[] allowedMonths = new Month[]{Month.JANUARY, Month.FEBRUARY, Month.MARCH, Month.APRIL, Month.MAY, Month.JUNE, Month.JULY, Month.AUGUST, Month.SEPTEMBER, Month.OCTOBER, Month.NOVEMBER, Month.DECEMBER};
+
+	@Override
+	protected Arbitrary<Month> arbitrary() {
```

Comment:
I suggest the following implementation

	@Override
	protected Arbitrary<Month> arbitrary() {
		List<Month> values = Arrays.asList(Month.class.getEnumConstants());
		final int indexMin = values.indexOf(min);
		final int indexMax = values.indexOf(max);

		return Arbitraries.integers()
						  .between(indexMin, indexMax)
						  .edgeCases(integerConfig -> integerConfig.includeOnly(indexMin, indexMax))
						  .map(values::get)
						  .filter(this::isInAllowedMonths);
	}

This will break one exhaustive generation test because this generation is more precise.

Reply:
It can even be further simplified if we drop `MonthArbitrary.includeOnly(..)` which is just a complicated way of saying `Arbitrary.of(..)`.